verilog/formatting/tree_unwrapper_test.cc

```diff
@@ -4118,6 +4118,26 @@ const TreeUnwrapperTestData kUnwrapPreprocessorTestCases[] = {
             L(0, {"endmodule"})),
     },
 
+    {
+	"Partitioning formal arguments in `define",
+	"`define FOO(BAR1, BAR2, BAR3)\n",
+	N(0, L(0, {"`define", "FOO", "("}),
+	  N(2,
+	    L(2, {"BAR1", ","}),
+	    L(2, {"BAR2", ","}),
+	    L(2, {"BAR3", ")", ""}))),
+    },
+
+    {
+	"Partitioning formal arguments in `define with body definition",
+	"`define FOO(BAR1, BAR2, BAR3) definition_body\n",
+	N(0, L(0, {"`define", "FOO", "("}),
+	  N(2,
+	    L(2, {"BAR1", ","}),
+	    L(2, {"BAR2", ","}),
+	    L(2, {"BAR3", ")", "definition_body"}))),
+    },
+
```

Comment:
Good so far.  More suggested test cases:

* two macro definitions in a row (combinations with/without definition bodies)
* a macro definition with at least one formal parameter with a default value

Reply:
Added more tests related to formal arguments.